src/main/java/org/cru/redegg/servlet/RedEggFilter.java

```diff
@@ -52,6 +52,7 @@ private void doFilter(HttpServletRequest request, HttpServletResponse response,
         throws IOException, ServletException {
         RecordingResponse recordingResponse = new RecordingResponse(response);
         RecordingRequest recordingRequest = new RecordingRequest(request, errorLog, getRecorder());
+        getRecorder().startMonitoringRequestForTimeliness();
```

Comment:
for this filter it doesn't appear that you ever stop monitoring the request

Reply:
Or is it a combination between starting here and ending the monitoring within `RedEggServletListener`

Reply:
that's right.
It's nice to stop monitoring as late as possible, so we can potentially catch problems further down the filter stack.

Offhand, I can't remember why I didn't start monitoring in the servlet listener as well, but I believe there was some reason.

Reply:
I can't find a good reason.
Perhaps I was mis-thinking that this would let the entity be available in stuck-request reports.
Or maybe I was thinking I was going to allow some non-http details to be recorded by the app (which would be reported in stuck thread reports), in a custom filter or something; but I didn't implement that. 

I'm going to change this.

Reply:
done